dojo_plugin/utils/__init__.py

```diff
@@ -143,6 +143,7 @@ def render_markdown(s):
         "b", "i", "strong", "em", "tt",
         "p", "br",
         "span", "div", "blockquote", "code", "pre", "hr",
+        "hide",
```

Comment:
Are we sure this line does anything? I believe this would allow "<hide>" elements, which I don't believe you want (or exist).

Reply:
it shouldn't have the "hide" tag
I went with using an attribute on p

Reply:
Ok, I removed, the hide tag, sorry about that.